PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,26 @@
-Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).
-
-When filling in a section please remove the help text and the above text.
+<!--
+Please make sure you've read the pull request section (http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) of our code contribution guidelines (http://docs.mattermost.com/developer/contribution-guide.html).
+-->
 
 #### Summary
-[A brief description of what this pull request does.]
+<!--
+A brief description of what this pull request does.
+-->
 
 #### Ticket Link
-[Please link the GitHub issue or Jira ticket this PR addresses.]
+<!--
+Please link the GitHub issue or Jira ticket this PR addresses.
+-->
 
 #### Checklist
-[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
+<!--
+Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.
+-->
 - [ ] Ran `make check-style` to check for style errors (required for all pull requests)
 - [ ] Ran `make test` to ensure unit tests passed
 - [ ] Ran `make flow` to ensure type checking passed
 - [ ] Added or updated unit tests (required for all new features)
 - [ ] All new/modified APIs include changes to the [JavaScript driver](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.js)
```

Comment:
I'm inclined to drop the checklist. To me it doesn't provide much value, as 1. - 3. are checked by CI and 4.-5. should be checked by the reviewers.

Thoughts on this?

Reply:
I also think there is not much sense to leave it as checklist. Nevertheless, it would be a nice to have as comment, as a friendly reminder, specially for those new to PRs in mattermost. Could end up as something like:
Remember to:
- Run `make check-style` ...
- ...
- ...

Reply:
:+1: for having a "keep this in mind list"

Reply:
Yep! Keep this in mind list would be nice.